components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,336 @@
++++
+title = "ServiceNow Integration"
+
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "ServiceNow Integration"
+    parent = "automate/reference"
+    identifier = "automate/reference/servicenow_integration.md ServiceNow Integration"
+    weight = 40
++++
+
+
+## Chef Automate - CMDB Integration App in ServiceNow
+
+This guide provides instructions to configure a customer's instance for enabling the population of the Configuration Management Database (CMDB) data within ServiceNow. Chef Automate manages the CMDB data from each node in the customer's estate and the compliance and configuration management status across the Enterprise.
+
+## Chef Automate and ServiceNow Integration
+
+The integration between a Chef Automate server and a ServiceNow instance requires the following components:
+
+- Chef Automate Scoped Application
+- Chef Automate Server
+
```

Comment:
We should have a note saying : The reader of this article should have fundamental understanding of ServiceNow ecosystem.

Reply:
done